### PR TITLE
Add guide on all things related to GitHub use

### PR DESCRIPTION
**Pull request by Cruikshanks**
Our principles push for us [Work in the open](https://github.com/DEFRA/dst-guides/blob/master/principles/work_in_the_open.md), which means publishing stuff to **GitHub**. However we don't actually have documented anywhere what the requirements are to access it, how we have structured it, and what we expect of users once they have access to our organisation on **GitHub**.

Part of the reason was that we had not formally agreed some of these decisions, but this has been rectified following a meeting of the core maintainers.

So this guide captures the decisions made, and forms a basis to collaborate and discuss how we want to use GitHub in the future.

### CODE REVIEW

**Comment by Cruikshanks** [comment]
@bensagar-ea and @davidblackburn I've pegged you initially to review this. I wanted to get it in before the dev get together next week, so you have a chance of announcing/discussing it then.

Apologies its a bit last minute, but squeezing it in before I dash off for 2 weeks! I believe it covers what we talked about. However if you think it needs to be changed could I ask you agree the changes between you and apply them in my absence? More than happy for you to just go ahead rather than wait for my return.

It being GitHub, I can always just raise a new PR if I don't agree with what you did 😈 !

File: github/README.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+- Teams represent all members of a service, however not all of them should have *Admin* access
+
+The administrator for each repo will be added as a **Collaborator**. Using GitHub's available functionality this is seen as the best solution
+
+- It avoids creating *Admin only* teams for each service
+- It helps distinguish who is the administrator for a repository
+
+<img src="collaborators_teams.png" alt="Collaborators & teams" style="width: 600px;"/>
+
+Also added as *Collaborators* will be any contractors or suppliers who require access to the repo. There access should be set to either *Read* or *Write*. Only if the repo is being maintained by a third party, such as a supplier will a collaborator be given *Admin* access.
```
Comment (bensagar-ea) [inline github/README.md]: Type: "There access..." should be "Their access..."

File: github/README.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+### Being an repository admin
```
Comment (bensagar-ea) [inline github/README.md]: Type: "an" should be "a"

File: github/README.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+If you are the administrator for a repository its your responsibility to ensure the repo has been setup and maintained in accordance with [New projects](new_projects.md) guide.
```
Comment (bensagar-ea) [inline github/README.md]: Typos: "its" should be "it's", "setup" should be "set up"

File: github/new_projects.md [hunk 4]
```diff
@@ -1,17 +1,17 @@
...
 - A name for the project
 - A description
 - Whether the repo is to be private or public
-- Team or teams to be assigned
+- Username of who will be its administrator
 
-**Do not create it under your own user account!** Though repo's can be transferred at a later date, it is simpler and clearer if the repo originates within the [Environment Agency](https://github.com/EnvironmentAgency) organisation on GitHub.
+**Do not create it under your own user account!** Though repo's can be transferred at a later date, it is simpler and clearer if the repo originates within the [Defra](https://github.com/Defra) organisation on GitHub.
 
 ## Add a licence and Readme
```
Comment (bensagar-ea) [inline github/new_projects.md]: The link to the README is confusing in this context as it takes you to the "home" page for these guides.

Maybe keep the text without the links and add an extra line with the links as examples?  Something like this:

> Before you push your first commit ensure you have added a LICENCE and a README to the project, and that the readme contains our standard sections covering **Contributing to this project** and **About the license**.
> The [LICENCE](/LICENSE) and [README](/README.md) files for these guides provide useful examples.

**Comment by Cruikshanks** [comment]
Did I mention I put this out in a rush? 😳 Thanks for catching all my grammar errors @bensagar-ea . Will get on and make the changes.

**Comment by Cruikshanks** [comment]
Thanks for the feedback @bensagar-ea . Hopefully all taken account of if you wouldn't mind double checking.

**Comment by bensagar-ea** [comment]
Thumbs up!

**Pull request merged**